indico/modules/rb/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -629,6 +629,34 @@ class BookRoomModal extends React.Component {
                     fprops.form.mutators
                   )}
               </Form>
+
+              {room.canUserViewInternalNotes && (
+                <Form
+                  id="book-notes-form"
+                  onSubmit={fprops.handleSubmit}
+                  style={{paddingTop: '1em'}}
+                >
+                  <Segment inverted styleName="booking-notes-section">
```

Comment:
Does this need to be in a separate form, or could it go inside `#book-room-form`?

Reply:
I would rather keep it separate.